inc/dirdefs.h

```diff
@@ -1,17 +1,9 @@
 #ifndef DIRDEFS_H
 #define DIRDEFS_H 1
 
-#include <dirent.h>         // for MAXNAMLEN
-
-/* Needed for Alpine Linux */
-#ifndef MAXNAMLEN
-#include <limits.h>
-#define MAXNAMLEN NAME_MAX
-#endif
-/* End Needed for Alpine Linux */
-
 #include <sys/types.h>      // for u_short, ino_t
 #include "lispemul.h"       // for LispPTR
+#include "locfile.h"        // for MAXNAMLEN
```

Comment:
Why not just include `<limits.h>` here as well? I am on my phone so don’t have full context readily available…

Reply:
I did that because `<limits.h>` will get you `NAME_MAX`, but not `MAXNAMLEN` and `"locfile.h"` has the `ifndef MAXNAMLEN` logic to define it as `NAME_MAX`, which I wanted to keep in just one place until we do the wholesale replacement of MAXNAMLEN with NAME_MAX (plus the logic for when NAME_MAX is not defined, such as on Solaris)

Reply:
Makes sense.